rplugin/python3/deoplete/sources/look.py

```diff
@@ -13,18 +14,36 @@ def __init__(self, vim):
 
         self.name = 'look'
         self.mark = '[look]'
-        self.min_pattern_length = 4
-        self.executable_look = self.vim.call('executable', 'look')
+
+        def get_neco_look_var(shortname, default):
+            return vim.vars.get('deoplete#neco_look#{}'.format(shortname), default)
+
+        self.min_pattern_length = get_neco_look_var('min_pattern_length', 4)
```

Comment:
you mean `deoplete#custom#set`, right? I actually just realized how similar the deoplete and denite interfaces are.